docs/troubleshooting/supported-browsers.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Supported Browsers List
+image: "/docs/images/docs-meta-cards/troubleshoot-card.png"
+displayed_sidebar: docs
+description: "Webpack 5 Issues | Documentation - Web3Auth"
```

Comment:
@Saviour1001 

the description says it's a `Web3pack 5 Issues`.

Reply:
remove `/docs/` from image (check recent change according to `vercel-fix`) and change the description.